addons/test_mail/tests/test_mail_template.py

```diff
@@ -135,17 +135,6 @@ def test_template_translation_partner_lang(self):
                          '<body><p>SpanishBody for %s</p> Spanish Layout para Spanish description</body>' % self.test_record.name)
         self.assertEqual(mail.subject, 'SpanishSubject for %s' % self.test_record.name)
 
-    def test_template_add_context_action(self):
```

Comment:
Why removing this test ?

Reply:
Indeed, I forgot to put it back with the context action feature :sweat_smile: